package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-loop",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
Yep, can do.  For the back-ported versions, is it okay to leave them?  The backporting strategy I've used automatically incorporates each new version into history, which would be lost if they were bumped after merge.  If you want, I can separate out the bumps into separate commits.